### PR TITLE
Added enableJsApi parameter for Youtube view helper

### PR DESCRIPTION
**Pull request by wtfred**
Hello,

Just a small PR to add the "enablejsapi" parameter in the Youtube View Helper in order to activate yhe Youtube Javascript Api (https://developers.google.com/youtube/player_parameters#enablejsapi)

Thanks for your extension :) !

### CODE REVIEW

File: Classes/ViewHelpers/Media/YoutubeViewHelper.php [hunk 1]
```diff
@@ -54,6 +54,7 @@ public function initializeArguments() {
 		$this->registerArgument('extendedPrivacy', 'boolean', 'Whether to use cookie-less video player.', FALSE, TRUE);
 		$this->registerArgument('hideControl', 'boolean', 'Hide video player\'s control bar.', FALSE, FALSE);
 		$this->registerArgument('hideInfo', 'boolean', 'Hide video player\'s info bar.', FALSE, FALSE);
+		$this->registerArgument('enableJsApi', 'boolean', 'Enable Youtupe API', FALSE, FALSE);
```
Comment (xf-) [inline Classes/ViewHelpers/Media/YoutubeViewHelper.php]: typo Youtupe

**Comment by cedricziel** [comment]
I really like this change - could you amend to the description I proposed (If you like it as well) and then squash it together with a commit message conforming to our CGL? ("[TASK] Add enableJsApi paramter to YouTubeViewHelper")

Thank you for the contribution! :)

**Comment by wtfred** [comment]
Hello,
Here it is, I hope i made it the way you ask :)

**Comment by cedricziel** [comment]
Almost - thank you so far :)

Would you please squash the commits together? If you don't know how to do it, it's the perfect moment to learn :)

**Comment by cedricziel** [comment]
PS: If not - no worries; I can take your commits and perform the magic for you or support you if you have any questions

**Comment by wtfred** [comment]
Never heard of "squash", i just looked into it and made some tries but it seems i'm not doing this right:

Here is what i did:
- Clone vhs repository from my account (i made the previous edition directly on github)
- git rebase -i head~2
- replace "pick" by "squash" in both commit lines
- save
- git commit --amend to edit the last commit message

But then i don't have the last modification of the description in the YoutubeViewHelper file... What did i do wrong ? ^^

**Comment by cedricziel** [comment]
Nothing, I guess :) Shouldnt it be `HEAD~3` because we have 3 commits in the list so far?

**Comment by cedricziel** [comment]
Oh by the way - you need to force push to the branch then to update the pull-request

**Comment by wtfred** [comment]
Ok, didn't had time today i check that tomorrow :)

**Comment by wtfred** [comment]
I can't squash properly, i keep having "Cannot 'squash' without a previous commit so i'll just close this PR and remake a new one ^^

**Pull request closed without merging**